azure-pipelines.yml

```
@@ -30,9 +30,13 @@ jobs:
       ./julia-$(JULIA_VERSION)/bin/julia -e 'using InteractiveUtils; versioninfo()'
       ./julia-$(JULIA_VERSION)/bin/julia --project=@. -e 'using Pkg; Pkg.instantiate()'
       ./julia-$(JULIA_VERSION)/bin/julia --project=@. -e 'using Pkg; Pkg.test()'
-      ./julia-$(JULIA_VERSION)/bin/julia -e 'using Test; @testset "system image" begin;@test include(".dev/systemimage/climate_machine_image.jl"); end;'
     displayName: 'Run the tests'
     continueOnError: true
+  - bash: |
+      set -o xtrace
+      ./julia-$(JULIA_VERSION)/bin/julia -e 'include(joinpath(".dev", "systemimage", "climate_machine_image.jl"))'
```

Should we just call the script?
```suggestion
      ./julia-$(JULIA_VERSION)/bin/julia .dev/systemimage/climate_machine_image.jl
```

---

yes probably, didn't know if we want to re-wrap this in a test macro (if this is the issue).